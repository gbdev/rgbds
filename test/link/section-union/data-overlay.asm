IF !DEF(SECOND)
	def DATA equs "ds 4"
ELSE
	def DATA equs "db $aa, $bb, $cc, $dd"
ENDC

SECTION UNION "overlaid data", ROM0
	DATA

	PURGE DATA
