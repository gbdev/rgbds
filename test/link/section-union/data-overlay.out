error: Section "overlaid data" is of type ROM0, which cannot be unionized
---
error: <stdin>(18):
    Cannot declare ROM sections as UNION
FATAL: <stdin>(18):
    Cannot create section "overlaid data" (1 error)
